x-pack/plugins/spaces/server/saved_objects/schemas.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { schema } from '@kbn/config-schema';
+
+export const SpacesSavedObjectSchemas = {
+  '8.7.0': schema.object({
+    description: schema.maybe(schema.string()),
+    initials: schema.string(),
+    color: schema.string(),
```

Comment:
__question:__ are these fields really required? Looking at the `x-pack/plugins/spaces/server/lib/space_schema.ts`, it seems they shouldn't be.

Reply:
😅 Ahh that makes sense...I was comparing against the UI. Fixed in latest commit.